src/Hl7.Fhir.Base/ElementModel/TypedElementOnSourceNode.cs

```diff
@@ -75,11 +78,12 @@ private TypedElementOnSourceNode(TypedElementOnSourceNode parent, ISourceNode so
             Definition = definition;
             InstanceType = instanceType;
             _settings = parent._settings;
+            Location = location;
         }
 
-        public ExceptionNotificationHandler ExceptionHandler { get; set; }
+        public ExceptionNotificationHandler ExceptionHandler { get; set; } = null!;
```

Comment:
Actually, this property *is* nullable I think?  It's possible to have no ExceptionHandler....